suite-common/connect-init/src/connectInitThunks.ts

```diff
@@ -45,7 +46,9 @@ export const connectInitThunk = createThunk(
         });
 
         TrezorConnect.on(UI_EVENT, ({ event: _, ...action }) => {
-            // dispatch event as action
+            if (action.type === 'ui-call_in_progress') {
```

Comment:
Hmmm maybe it should be `DEVICE_EVENT` 🤔

Reply:
I agree that `DEVICE_EVENT` sounds a bit better.

Reply:
Also, I thought it's more obvious not to edit Connect events here in `connectInitThunks` but pass them directly as redux actions whenever possible. So I tried to catch this event in Suite middleware and transform it to `SUITE.LOCK_DEVICE` there, but then it probably wouldn't work in native, because they don't use the same middlewares? So I'm not sure about that now.

Reply:
Sure, middleware sounds like a place to put it. I just don't like to use middleware if there is only a single use of something. In that case, it feels more readable without it. 

🤔 but there are also `ui-*` device-related events

```
'ui-device_needs_backup'
'ui-request_passphrase_on_device'
'ui-request_pin'
```

Reply:
> I just don't like to use middleware if there is only a single use of something

Yes, I agree, that's why my first idea was to directly replace `SUITE.LOCK_DEVICE` with `UI.CALL_IN_PROGRESS` everywhere. But that would be a bigger change.